_content/booklets/mettabhavana_anandajoti.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Mettabhāvanā (Universal version) Friendliness Meditation"
+authors:
+  - anandajoti
+external_url: "https://ancient-buddhist-texts.net/Texts-and-Translations/Short-Pieces/Mettabhavana-2.htm"
+drive_links:
+  - "https://drive.google.com/file/d/1knSztT3Wx2BO9feh5IFBOUypoRgO8wiX/view?usp=drive_link"
+course: sri-lankan
+tags:
+  - pali-canon
```

Comment:
Not Pali Canon (even though it's written in Pali). It's loosely based on the Vsm.  Let's tag with `brahmavihara`